README.md

```diff
@@ -105,9 +105,30 @@ steps:
 ```
 - Flows are separated by `---` as standardized by YAML syntax to separate documents
 - It is possible to call a flow using either the **id** of the flow or the **index** of the flow (one indexed)
-- Connections opened inside an include will remain there. Therefore consequent packets (after the include) will use a connection defined in main flow.
+- Connections opened inside an include will remain there. Therefore, consequent packets (after the include) will use a connection defined in main flow.
 	> Looking at the example, the packet between *includes* will use the connection established in port 23.
-          
+
+### Parallel execution
+
+Multiple flows which needs to run in parallel?
```

Comment:
need*

Reply:
Is a little weird the way of presenting it (by question and then answer the question), I think it adds unnecessary noise (time of reading) even though it might make reading a little less boring. Maybe just: If you want to run multiple parts of a flow in parallel (the order of such parts is not relevant for flow execution) then you can use the parallel step: